trillian/ctfe/instance_test.go

```diff
@@ -179,28 +178,25 @@ func TestSetUpInstance(t *testing.T) {
 	}
 }
 
-func equivalentTimes(a *time.Time, b *timestamp.Timestamp) bool {
+func equivalentTimes(a *time.Time, b *timestamppb.Timestamp) bool {
 	if a == nil && b == nil {
 		return true
 	}
 	if a == nil {
 		// b can't be nil as it would have returned above.
 		return false
 	}
-	tsA, err := ptypes.TimestampProto(*a)
-	if err != nil {
-		return false
-	}
-	return ptypes.TimestampString(tsA) == ptypes.TimestampString(b)
+	tsA := timestamppb.New(*a)
+	return tsA.AsTime().Format(time.RFC3339Nano) == b.AsTime().Format(time.RFC3339Nano)
```

Comment:
Would checking `b.AsTime() == *a` suffice, instead of the last 2 lines?

Reply:
I have no idea, all I know is that this is semantically identical (it's more or less the inlined implementation of `ptypes.TimestampString`).

Reply:
Since this is a testing code, you could give it a try and see if tests pass?

Reply:
I'm worried of weakening the test, which wouldn't be revealed by a failure...

Reply:
Eh? Checking for exact equality of `x` and `y` seems stronger than checking that formatted string `x.Format() == y.Format()`? So you could make the test stronger, not weaker? I'm just proposing to drop some redundant back and forth conversions between `Time` and proto.